extraTs/pattern/equilateral.ts

```diff
@@ -0,0 +1,14 @@
+//#39515 Equilateral pattern
+const createDimondShape = (size)=>{
+    for(var i=1;i<=size;i++){
+       for(var s=size-1;s>=i;s--){
+          process.stdout.write(" ");
+       }
+       for(var j=1;j<=i;j++){
+          process.stdout.write("* ")
+       }
+       console.log();
+    }
+}
+
```

Comment:
Add input type and return type of function

Reply:
done